### ISSUE TITLE
adding Segwit on Litecoin to your wallets?

### ISSUE THREAD

**Issue by ghost** [LedgerHQ/ledger-nano-s#18]
When will you add Segwit on Litecoin to your wallets now when it´s activated?

**Comment by kosciej** [comment]
As far I know it's already added in Bitcoin/Altcoins chrome app. I can see my litecoins on segwit address in this app.

**Comment by jonathancross** [comment]
@briannyeko Have you had a chance to take a look at the Chrome app?

**Comment by ghost** [comment]
Yes, I have been using your Chrome app for over a year, but a chrome app is
not the same as a "electron app which you install independently from
chromes webstore"
